express_relay/testing/tests/initialize.rs

```diff
@@ -0,0 +1,57 @@
+use express_relay::state::FEE_SPLIT_PRECISION;
+use solana_sdk::signer::Signer;
+use testing::{express_relay::helpers::get_express_relay_metadata, helpers::assert_custom_error, setup::{setup, SetupParams}};
+
+#[test]
+fn test_initialize() {
+    let split_protocol_default: u64 = 4000;
+    let split_relayer: u64 = 2000;
+
+    let setup_params = SetupParams {
+        split_protocol_default,
+        split_relayer,
+    };
+    let setup_result = setup(setup_params).expect("setup failed");
+
+    let express_relay_metadata = get_express_relay_metadata(setup_result.svm);
+
+    assert_eq!(express_relay_metadata.admin, setup_result.admin.pubkey());
+    assert_eq!(express_relay_metadata.relayer_signer, setup_result.relayer_signer.pubkey());
+    assert_eq!(express_relay_metadata.fee_receiver_relayer, setup_result.fee_receiver_relayer.pubkey());
+    assert_eq!(express_relay_metadata.split_protocol_default, split_protocol_default);
+    assert_eq!(express_relay_metadata.split_relayer, split_relayer);
+}
+
+#[test]
+fn test_initialize_fail_high_split_protocol() {
+    let split_protocol_default: u64 = FEE_SPLIT_PRECISION + 1;
+    let split_relayer: u64 = 2000;
+
+    let setup_params = SetupParams {
+        split_protocol_default,
+        split_relayer,
+    };
+    let setup_result = setup(setup_params);
+
+    match setup_result {
+        Ok(_) => panic!("expected setup to fail"),
+        Err(err) => assert_custom_error(err, 0, 6000)
```

Comment:
these assertions are a bit fragile and unreadable. We can use something like this `express_relay::error::ErrorCode::FeeSplitLargerThanPrecision.into()` instead of `6000`